test/local/test_cli.py

```diff
@@ -679,6 +679,10 @@ def test_cmd_set(self):
                                          "--detached",
                                          '-y']))
 
+    def test_cmd_foreach(self):
```

Comment:
the test is a bit weak, it does not validate the different command-line options. Not sure if they can easily be validated.